samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -58,12 +51,12 @@
         </tr>
     </thead>
     <tbody>
-        @foreach (var node in pagedNodeList.Results)
+        @foreach (var node in PagedNodeList.Results)
         {
             <tr>
                 <td>@node.NodeName</td>
                 <td>@node.Id</td>
-                @if (node.Children == true)
+                @if ((node.Children == true) && (node.NodeClass != NodeClass.Method))
```

Comment:
>(node.Children == true) [](start = 21, length = 23)

do you really need to do it comparing a bool with a bool. Is Children a bool?

Reply:
removed comparison